services/QuillConnect/app/libs/flagArray.js

```diff
@@ -0,0 +1,11 @@
+export default (flag) => {
+  let flagArray = ['production']
+  if (flag === 'alpha') {
+    flagArray = ['alpha', 'beta', 'production']
+  } else if (flag === 'beta') {
+    flagArray = ['beta', 'production']
+  } else if (flag === 'archived') {
+    flagArray = ['archived', 'alpha', 'beta', 'production']
+  }
+  return flagArray
+}
```

Comment:
I wonder if it might be worth having a function in this file that does the comparison instead of checking `includes` in any other file that needs to know if the lesson and question flag types match up.

Reply:
That's a good idea.